Dockerfile

```diff
@@ -1,22 +1,11 @@
 FROM alpine
 
 RUN apk --update add go git \
-  && ORG_PATH="github.com/Financial-Times/" \
-  && REPO_PATH="${ORG_PATH}ft-vulcan" \
   && export GOPATH=/gopath \
-  && export PATH=$PATH:$GOPATH/bin \
-  && mkdir -p $GOPATH/src/${ORG_PATH} \
-  && ln -s ${PWD} $GOPATH/src/${REPO_PATH} \
-  && go get github.com/mailgun/vulcand \
-  # checkout older version of vulcan \
-  && cd $GOPATH/src/github.com/mailgun/vulcand \
-  && git checkout 791285c97cbf28f8eac1ef7222e103990c2d0b08 \
-  && go get github.com/Financial-Times/vulcan-session-auth/sauth \
-  && go get github.com/mailgun/vulcand/vbundle \
-  && go get github.com/mailgun/log \
-  && cd $GOPATH/src/${REPO_PATH} \
-  && vbundle init --middleware=github.com/Financial-Times/vulcan-session-auth/sauth \
-  && go build -o vulcand \
+  && go get github.com/Financial-Times/vulcand \
+  && cd $GOPATH/src/github.com/Financial-Times/vulcand \
+  && git checkout daaa02dfc0d1d5e6f34440825008a743cc5b442f \
```

Comment:
The hash won't change, we that PR gets merged, the hash becomes the HEAD on master. I can then merge both, get a new version in dockerhub and use that to test in xp/pre-prod.